_sections/80-media.md

```diff
@@ -30,3 +30,12 @@ The `<itext>` element described in the [languages](#languages) section can also
 By default, itext "image" values are not clickable. However, if you also include a "big-image", the image displayed by "image" will be clickable and will display a pannable, zoomable view of the file specified by "big-image". The user interface must provide a way to go back to the form after opening a "big-image". Specifying "big-image" alone has no effect, you must always include "image".
 
 Files referenced by "image" and "big-image" may be the same; however, for performance reasons, it is recommended to create smaller thumbnail images to be referenced by "image".
+
+#### Quality Levels
+
+Audio quality level can be specified using the `odk:quality` [bind attribute](#bind-attributes). Valid quality values are:
+
+* `normal` Should provide a "good enough" quality for most use cases
```

Comment:
I wanted to just document the intent of each quality level rather than going into detail here as it might be that different clients would need to use different codec etc.

Reply:
We could also link to the Collect docs here, but I'm not sure people would ever look at this while building a form. Thinking maybe it's best to avoid?

Reply:
I agree that we should not link to a specific client's docs from the spec (we've made an exception for `audit` but if other clients implement it we would redress it).

Would be good to see what @MartijnR thinks about this. Another thing we could do is document file types because it could be hard for downstream tools if clients are not consistent.

Reply:
I would be fine with adding some detail here. How about file types and adding the codec with parameters (that Collect uses) as an example? 

"in m4a format (e.g. using AAC, mono, 32kHz, 24kbps)"